.github/workflows/build-assets-compilation.yml

```diff
@@ -23,6 +23,11 @@ on:
         default: '-v --env=root'
         required: false
         type: string
+      PHP_VERSION:
+        description: PHP version with which the coding standard analysis is to be executed.
+        default: 7.4
+        required: false
+        type: string
```

Comment:
Please move this block to the very top of the inputs to align it with other workflows.